test/models/location_test.rb

```diff
@@ -9,4 +9,9 @@
 require 'test_helper'
 
 class LocationTest < ActiveSupport::TestCase
+  test 'should create a rescan runner' do
+    assert_difference('RescanRunner.count') do
```

Comment:
```suggestion
    assert_difference('RescanRunner.count', 1) do
```
I know that this is the default, but I prefer to be explicit about it, personally (feel free to disagree).

Reply:
I don't care that much so i've updated this. Can you check if there is a rubocop cop for this and enable it? It's more important to me that this is consistent